modules/gapi/src/compiler/gstreaming_priv.hpp

```diff
@@ -46,6 +46,7 @@ class GAPI_EXPORTS GStreamingCompiled::Priv
     void start();
     bool pull(cv::GRunArgsP &&outs);
     bool pull(cv::GOptRunArgsP &&outs);
+    std::tuple<bool, cv::util::variant<cv::GRunArgs, cv::GOptRunArgs>> pull();
```

Comment:
I believe there was already `pull(void)` introduced by @TolyaTalamanov back in the early Python binding development time - which allocated its outputs which are not optional. Is it gone now? Is it replaced by this version now?

Reply:
No, there was `GStreamingCompiled::pull` not `Priv::pull`.